scripts/setup-olm.sh

```diff
@@ -89,6 +89,10 @@ then
     fi
 else
     OLM_NS=openshift-marketplace
+    # Delete the custom scc if it already exists
+    kubectl delete scc anyuid-extra || :
+    # Create the custom scc
+    $SCRIPT_DIR/create-custom-scc.sh
```

Comment:
That step should be done after we install the operator in a specific namespace as the sa is namespace scoped. We cannot run "oc adm policy add-scc-to-user" if don't know yet the namespace where the operator will be installed. But we could write a small script that will have the namespace as argument to do that.

Reply:
I guess it would need to be part of each e2e test then.  Is there a way to override the scc in the csv at run time?  That way we can run the e2e tests with the relaxed scc without any change to the test.

Reply:
by relaxed scc do you mean privileged? Or is that you don't want to run e2e tests with privileged?

Reply:
Even if we do that pods that would be created would  run with privileged. If I am mistaking deploy-olm.sh is just for e2e tests right? Couldn't we add "oc adm policy add-scc-to-user" over there to bind the serviceaccount to the custom scc after checking whether we are in openshift?

Reply:
That may work too.  I just think it should be easy to run the e2e tests with anyuid-extra.  That is the mode we must run in for all of the tests to be successful.